tracing-opentelemetry/CHANGELOG.md

```diff
@@ -0,0 +1,45 @@
+# 0.4.0 (May 12, 2020)
+
+### Added
+
+- `tracing_opentelemetry::layer()` method to construct a default layer.
+- `OpenTelemetryLayer::with_sampler` method to configure the opentelemetry
+  sampling behavior.
+- `OpenTelemetryLayer::new` method to configure both the tracer and sampler.
+
+### Changed
+
+- `OpenTelemetrySpanExt::set_parent` now accepts a reference to an extracted
+  parent `Context` instead of a `SpanContext` to match propagators.
+- `OpenTelemetrySpanExt::context` now returns a `Context` instead of a
+  `SpanContext` to match propagators.
+- `OpenTelemetryLayer::with_tracer` now takes `&self` as a parameter
+- Upgrade to `v0.5.0` of `opentelemetry`.
```

Comment:
Can we call out the breaking changes explicitly? For example, see https://github.com/tokio-rs/tracing/blob/master/tracing-subscriber/CHANGELOG.md#breaking-changes

Reply:
👍